client/ayon_core/addon/base.py

```diff
@@ -411,82 +411,56 @@ def _load_addons_in_core(
 ):
     # Add current directory at first place
     #   - has small differences in import logic
-    hosts_dir = os.path.join(AYON_CORE_ROOT, "hosts")
     modules_dir = os.path.join(AYON_CORE_ROOT, "modules")
+    if not os.path.exists(modules_dir):
+        log.warning((
+            "Could not find path when loading AYON addons \"{}\""
+        ).format(modules_dir))
+        return
 
-    for dirpath in {hosts_dir, modules_dir}:
-        if not os.path.exists(dirpath):
-            log.warning((
-                "Could not find path when loading AYON addons \"{}\""
-            ).format(dirpath))
-            continue
-
-        is_in_modules_dir = dirpath == modules_dir
-        ignored_filenames = set()
-        if is_in_modules_dir:
-            ignored_filenames = set(IGNORED_DEFAULT_FILENAMES)
-
-        for filename in os.listdir(dirpath):
-            # Ignore filenames
-            if filename in IGNORED_FILENAMES or filename in ignored_filenames:
-                continue
+    ignored_filenames = set(IGNORED_DEFAULT_FILENAMES)
```

Comment:
This variable could just be a `set` on the module level constant already looking at the usages?

Same for `IGNORED_FILENAMES` maybe?